### ISSUE TITLE
Support for rails 4.2.1

### ISSUE THREAD

**Issue by Danielpk** [carlosantoniodasilva/i18n_alchemy#36]
I trying to update my rails version to 4.2.1 but it looks like i18n_alchemy isnt compatible with rails `activesupport  =<  4.2.0` is that right?

My Gemfile
```
gem 'i18n_alchemy', github: 'carlosantoniodasilva/i18n_alchemy', branch: 'master'
gem 'rails', '4.2.1'
```

What i doing:
```
$ bundle update rails
Fetching gem metadata from https://rubygems.org/..........
Fetching additional metadata from https://rubygems.org/..
Resolving dependencies...
Bundler could not find compatible versions for gem "activesupport":
  In Gemfile:
    i18n_alchemy (>= 0) ruby depends on
      activesupport (<= 4.2.0, >= 3.2.0) ruby

    rails (= 4.2.1) ruby depends on
      activesupport (4.2.1)
```

**Comment by carlosantoniodasilva** [comment]
You're correct, we were locking to `<= 4.2.0`, which does not allow for `4.2.1`. I've updated it locally but got some failures with 4.2.1 which I'll have to look at, so it will take a bit of time for the fix to come.

Thanks for reporting! :)

**Comment by martinsefcik** [comment]
any updates ?

**Comment by carlosantoniodasilva** [comment]
Sorry, I haven't had a chance to look back at this, if anyone has time it'd be appreciated, otherwise I'll try to do it in the following days/weeks.

**Comment by Danielpk** [comment]
@carlosantoniodasilva i tried to take a look but i couldnt find any solution. If you have any tip that can help i will try again...

**Comment by carlosantoniodasilva** [comment]
Fixed by #38, let us know if you're still seeing any issue. Thanks!

**Comment by carlosantoniodasilva** [comment]
Just released v0.2.2.
